Sources/RNLibreToolsiOS13/NFCOperations/NFCReadFramOperation.swift

```diff
@@ -15,7 +15,7 @@ final class NFCReadFramOperation: NFCAbstractOperation {
         try await sensor.scanHistory(tag: tag)
         logger.info(data.hexDump(header: "NFC: did read \(data.count / 8) FRAM blocks:", startBlock: start))
         try await securityManager.passPostSecurityChallengedIfNeeded(data: data)
-        try sensor.detailFRAM()
+        let sensInfo = try sensor.detailFRAM()
```

Comment:
This can be moved to `convertToReadFramResponse` since you need it there to convert to react native